src/openfhe.jl

```diff
@@ -74,28 +70,124 @@ end
 
 """
     init_rotation!(context::SecureContext{<:OpenFHEBackend}, private_key::PrivateKey,
-                   shifts)
+                   shape::NTuple{N, Integer}, rotation_index...)
 
-Generate rotation keys for use with `OpenFHE.EvalRotate` using the `private_key` and for the
-rotation indexes in `shifts`. The keys are stored in the given `context`.
-Positive shift defines rotation to the right, e.g. a rotation with a shift 1:
-[1, 2, 3, 4] -> [4, 1, 2, 3].
-Negative shift defines rotation to the left, e.g. a rotation with a shift -1:
-[1, 2, 3, 4] -> [2, 3, 4, 1].
-
-Note: Here, indexes stored in `shifts` have reversed sign compared to rotation indexes used in
-OpenFHE.
+Generate all required rotation keys for use with `circshift` for the
+rotation indexes in `rotation_index` using the `private_key`. The keys are stored in the given `context`.
 
 See also: [`SecureContext`](@ref), [`OpenFHEBackend`](@ref), [`PrivateKey`](@ref)
 """
 function init_rotation!(context::SecureContext{<:OpenFHEBackend}, private_key::PrivateKey,
-                        shifts)
+                        shape::NTuple{N, Integer}, rotation_index...) where N
```

Comment:
These `rotation_index`es are the same values as will be later passed to `circshift`, right? Maybe just call them `shifts` for consistency here, and later refer to them as "rotation indices" once they are converted to the actual cihpertext rotation values?